tests/sbiutils_test.py

```diff
@@ -406,30 +407,113 @@ def test_kde(bandwidth, transform, sample_weights):
 
 
 @pytest.mark.parametrize(
-    "z_x", [True, False, None, "none", "independent", "structured"]
+    "z_x",
+    [
+        True,
+        False,
+        None,
+        "none",
+        "independent",
+        "structured",
+        "transform_to_unconstrained",
+        "invalid_value",
+    ],
+)
+@pytest.mark.parametrize(
+    "z_theta",
+    [
+        True,
+        False,
+        None,
+        "none",
+        "independent",
+        "structured",
+        "transform_to_unconstrained",
+        "invalid_value",
+    ],
+)
+def test_z_score_parser(z_x, z_theta):
+    if z_theta == "invalid_value":
+        with pytest.raises(ValueError, match="Invalid z-scoring option."):
+            z_score_parser(z_theta)
+
+    if z_x == "invalid_value":
+        with pytest.raises(ValueError, match="Invalid z-scoring option."):
+            z_score_parser(z_x)
+
+    if z_x != "invalid_value" and z_theta != "invalid_value":
+        result_x = z_score_parser(z_x)
+        result_theta = z_score_parser(z_theta)
+
+        assert result_x is not None, f"z_score_parser({z_x}) returned None"
+        assert result_theta is not None, f"z_score_parser({z_theta}) returned None"
+
+
+@pytest.mark.parametrize(
+    "z_x",
+    [
+        True,
+        False,
```

Comment:
these are deprecated, please remove them from the test, or add them as `pytest.param(True, raises=Userwarning, match="Boolean flag for z-scoring is deprecated")` (please double check syntax). thanks! 🙏